docs/tutorials/aws/AIConnectorHelper.ipynb

```diff
@@ -0,0 +1,1069 @@
+    "                }\n",
+    "            ]\n",
+    "        }\n",
+    "\n",
+    "        print('Step 2: Configure IAM role in OpenSearch')\n",
+    "        print('Step 2.1: Create IAM role for Signing create connector request')\n",
+    "        if not self.role_exists(create_connector_role_name):\n",
+    "            create_connector_role_arn = self.create_iam_role(create_connector_role_name, trust_policy, inline_policy)\n",
+    "        else:\n",
+    "            print('role exists, skip creating')\n",
+    "            create_connector_role_arn = self.get_role_arn(create_connector_role_name)\n",
+    "        #print(create_connector_role_arn)\n",
+    "        print('----------')\n",
+    "\n",
+    "        # 2.2 Map backend role\n",
+    "        print(f'Step 2.2: Map IAM role {create_connector_role_name} to OpenSearch permission role')\n",
+    "        self.map_iam_role_to_backend_role(create_connector_role_arn)\n",
+    "        print('----------')\n",
+    "\n",
+    "        # 3. Create connector\n",
+    "        print('Step 3: Create connector in OpenSearch')\n",
+    "        # When you create an IAM role, it can take some time for the changes to propagate across AWS systems.\n",
+    "        # During this time, some services might not immediately recognize the new role or its permissions.\n",
+    "        # So we wait for some time before creating connector.\n",
+    "        # If you see such error: ClientError: An error occurred (AccessDenied) when calling the AssumeRole operation\n",
+    "        # you can rerun this function.\n",
+    "\n",
+    "        # Wait for some time\n",
+    "        time.sleep(sleep_time_in_seconds)\n",
+    "        payload = create_connector_input\n",
+    "        payload['credential'] = {\n",
+    "            \"roleArn\": connector_role_arn\n",
+    "        }\n",
+    "        connector_id = self.create_connector(create_connector_role_name, payload)\n",
+    "        #print(connector_id)\n",
+    "        print('----------')\n",
+    "        return connector_id"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cb94f44d",
+   "metadata": {},
+   "source": [
+    "# 0. Create helper"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "3e50f513",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "region = '...' # set your AWS region, for example us-west-2\n",
+    "opensearch_domain_name = '...' # set your AWS OpenSearch domain name\n",
+    "opensearch_domain_username = '...' # set your AWS OpenSearch domain admin username\n",
+    "opensearch_domain_password = '...' # set your domain password\n",
+    "\n",
+    "aws_user_name = '...' # set your AWS IAM user name, not IAM user ARN. \n",
+    "            # To avoid permission issue and quick start, you can use user whith AdministratorAccess policy\n",
+    "            # Configure this user's access key and secret key in ~/.aws/credential \n",
+    "\n",
```

Comment:
Can we provide an example here in ~/.aws/credential just to inform those people who are not familiar with the IAM Role permissions? Like this:
[You IAM Role Name]
aws_access_key_id = YOUR_ACCESS_KEY_ID
aws_secret_access_key = YOUR_SECRET_ACCESS_KEY

Reply:
done